requirements/app/test.txt

```diff
@@ -4,11 +4,13 @@ pytest-timeout ==2.1.0
 pytest-cov ==4.0.0
 pytest-doctestplus >=0.9.0
 pytest-asyncio ==0.20.3
+pytest-random-order ==1.1.0
 pytest-rerunfailures <=11.1.2
+
 playwright ==1.30.0
-httpx
+httpx <=0.24.0
```

Comment:
My previous comment also applies to all other `<=` or `>=` in the different `test.txt` files. Maybe you prefer to do it in a separate PR. Up to you

Reply:
```suggestion
httpx ==0.24.0
```